src/shared/user-record-show.c

```diff
@@ -132,7 +132,23 @@ void user_record_show(UserRecord *hr, bool show_full_group_info) {
                         break;
                 }
 
-                printf(" Password OK: %syes%s\n", ansi_highlight_green(), ansi_normal());
+                if (strv_isempty(hr->hashed_password)) {
+                        printf(" Password OK: %sno%s (none set)\n", ansi_highlight(), ansi_normal());
+                        break;
+                }
+                if (strv_contains(hr->hashed_password, "")) {
+                        printf(" Password OK: %sno%s (empty set)\n", ansi_highlight_red(), ansi_normal());
+                        break;
+                }
+                bool has_valid_passwords = false;
+                char **p;
+                STRV_FOREACH(p, hr->hashed_password)
+                        if (!hashed_password_is_locked_or_invalid(*p)) {
+                                has_valid_passwords = true;
+                                printf(" Password OK: %syes%s\n", ansi_highlight_green(), ansi_normal());
+                }
```

Comment:
due to false indentation this closing bracket suggests it was the closing bracket for the STRV_FOREACH(), but it isn't, it's the one for the `if` expression...

please fix indentation.